use-cases/mlflow-inference-pipeline/README.md

```diff
@@ -0,0 +1,225 @@
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Database 'mlflowdb' creation verified", "timestamp": 1744226226.1060863, "level": "INFO", "runtime": 6815.025777}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Connector closed", "timestamp": 1744226226.1147602, "level": "INFO", "runtime": 6823.699854}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Database 'mlflowdb' creation successful.", "timestamp": 1744226226.1149092, "level": "INFO", "runtime": 6823.848818}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Connection pool created for database 'mlflowdb'", "timestamp": 1744226226.1501238, "level": "INFO", "runtime": 6859.063597}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Granting ALL privileges on schema 'public' of database 'mlflowdb' to user 'wi-xxxx-user@xxxx.iam'", "timestamp": 1744226226.150347, "level": "INFO", "runtime": 6859.286788}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Connecting to database 'mlflowdb' as user ''wi-xxxx-user@xxxx.iam'", "timestamp": 1744226226.1509507, "level": "INFO", "runtime": 6859.890375}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Successfully granted ALL privileges on schema 'public' of database 'mlflowdb' to user 'wi-xxxx-user@xxxx.iam'", "timestamp": 1744226226.8060656, "level": "INFO", "runtime": 7515.005285}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Connector closed", "timestamp": 1744226226.8080332, "level": "INFO", "runtime": 7516.973029}
+{"name": "__main__", "thread": 135677854571392, "threadName": "MainThread", "processName": "MainProcess", "process": 1, "message": "Permissions granted to user 'wi-xxxx-user@xxxxx.iam' on database 'mlflowdb'.", "timestamp": 1744226226.8089216, "level": "INFO", "runtime": 7517.86128}
+```
+
+- ML flow backend store
+
+  Mlflow needs an artifact store to store the mlflow deployment artifacts. A GCS
+  bucket has already been created as part of MLP Playground.
+
+  Validate that this GCS bucket exists in your project.
+
+  ```shell
+  gcloud storage buckets describe gs://${MLP_MLFLOW_ARTIFACT_LOCATION}
+  ```
+
+## Build the MLflow container image:
+
+- Configure build job
+
+```shell
+ cd mlflow
```

Comment:
It would probably be easier to `cd` into the mlflow directory once and then run commands out of there instead of `cd` and `cd -` back and forth.